apps/agreement/test/agreement/agreement_challenge.js

```diff
@@ -29,7 +29,7 @@ contract('Agreement', ([_, submitter, challenger, someone]) => {
       const itCanChallengeActions = () => {
         context('when the challenger has permissions', () => {
           const stake = false // do not stake challenge collateral before creating challenge
-          const arbitrationFees = false // do not approve arbitration fees before creating challenge
+          const arbitratorFees = false // do not approve arbitration fees before creating challenge
```

Comment:
Not your fault, but we should probably do a find and replace
```suggestion
          const arbitratorFees = false // do not approve arbitrator fees before creating challenge
```

Reply:
I saw this, but as they were comments and they are kind of synonyms I didn’t think it was important. I changed only the variable names because I had some problems with the different namings.